### ISSUE TITLE
Component name read twice by VoiceOver on storefront

### ISSUE THREAD

**Issue by Febakke** [digdir/designsystemet#3216]
On our component overview the names of the components are read twice by VoiceOver. Every link also has a H2 header. The read twice part should be removed, but im not sure what is best for headers on this page. 

[Storefront: components](https://pr-3215.designsystemet.no/komponenter)

````
<a class="ComponentCard_card__xjfpP ds-focus" href="https://next.storybook.designsystemet.no/?path=/docs/komponenter-alert--docs">
<img alt="Alert" class="ComponentCard_image__TloXz" src="/img/component-previews/Alert.svg">
<h2 class="ds-heading ComponentCard_title__u_czK" data-size="xs">Alert</h2>
</a>
````

Im guessing its the alt text for the image combined with heading. Maybe leave `alt=""`
